pkg/operations/deletevm.go

```diff
@@ -75,7 +75,7 @@ func CleanDeleteVirtualMachine(az armhelpers.ACSEngineClient, logger *log.Entry,
 		}
 
 		logger.Infof("deleting blob: %s/%s", vhdContainer, vhdBlob)
-		if err = as.DeleteBlob(vhdContainer, vhdBlob); err != nil {
+		if err = as.DeleteBlob(vhdContainer, vhdBlob, nil); err != nil {
```

Comment:
Thanks a lot @mick0601 . Last request :Can we pass an empty DeleteOptionBlob struct pointer instead of nil here?

Reply:
Yes, I will make that change in a real quick. Thanks again for the help, Tariq